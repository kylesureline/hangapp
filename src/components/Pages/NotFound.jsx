import React from "react";
import DocumentTitle from "react-document-title";

export const NotFound = () => (
  <DocumentTitle title={`${process.env.REACT_APP_NAME} | Page Not Found`}>
    <main>404</main>
  </DocumentTitle>
);
