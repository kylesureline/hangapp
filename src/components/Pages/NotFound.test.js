import React from "react";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";
import { NotFound } from "./NotFound";

describe("<NotFound />", () => {
  it("matches snapshot", () => {
    expect(toJson(mount(<NotFound />))).toMatchSnapshot();
  });
});
