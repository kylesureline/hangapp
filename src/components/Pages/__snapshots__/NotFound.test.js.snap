// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> matches snapshot 1`] = `
<NotFound>
  <SideEffect(DocumentTitle)
    title="Hangapp | Page Not Found"
  >
    <DocumentTitle
      title="Hangapp | Page Not Found"
    >
      <main>
        404
      </main>
    </DocumentTitle>
  </SideEffect(DocumentTitle)>
</NotFound>
`;
